app/classes/query/modules/sequence.rb

```diff
@@ -131,6 +131,12 @@ def first
     new_self
   end
 
+  def first_id
```

Comment:
1. Is this dead code? The only call I could find was QueryTest, line 428
2. If it's not dead, will this work:

```suggestion
    result_ids.&first
```